redhat-appstudio/service-provider-integration-operator
# Add permissions for end users to edit/view spiaccesstokendataupdate

### What does this PR do?
Add permissions for end-users to edit/view spiaccesstokendataupdate

### Screenshot/screencast of this PR

<img width="1669" alt="Знімок екрана 2022-05-17 о 16 28 01" src="https://user-images.githubusercontent.com/1614429/168822215-d0c6b790-822a-4e01-8907-2df7109d85e4.png">


### What issues does this PR fix or reference?
https://issues.redhat.com/browse/SVPI-121


### How to test this PR?
See https://github.com/redhat-appstudio/service-provider-integration-oauth/pull/88
